ChangeLog.md

```diff
@@ -4,6 +4,11 @@
 
 RabbitMQ Java client dependency has been updated to the final version of `4.2.0`.
 
+When a TLS/SSL certificate path and password is provided for a PKCS12 keystore, those certificates from that keystore will now also be used (by default) for the TLS/SSL trust instead of the `NullTrustManager`.  
```

Comment:
Please add a change log entry for this (and credit yourself + Elastic).

Reply:
oops put this in the wrong spot, will fix.